manifests/WiXToolset/WiXToolset/3.11.2.yaml

```diff
@@ -0,0 +1,17 @@
+Id: WiXToolset.WiXToolset
+Version: 3.11.2
+Name: WiX Toolset
+Publisher: WiX Toolset
+License: Copyright (c) .NET Foundation and contributors
+LicenseUrl: https://github.com/wixtoolset/wix3/blob/master/LICENSE.TXT
+Tags: msi, wix, installer
+Description: The WiX toolset lets developers create installers for Windows Installer, the Windows installation engine.
+Homepage: https://wixtoolset.org/
+Installers:
+  - Arch: x86
+    Url: https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311.exe
+    Sha256: 32BB76C478FCB356671D4AAF006AD81CA93EEA32C22A9401B168FC7471FECCD2
+    InstallerType: exe
+    Switches:
+      Silent: /q
+      SilentWithProgress: /q
```

Comment:
Silent with progress is "/passive".  "/q" doesn't show progress

Reply:
Thanks, fixed it.

Reply:
@TheQuazman "/passive" does not seem to work for this installer. The UI is displayed but it shows a generic error and installation does not start.